localstack/aws/api/s3/__init__.py

```diff
@@ -232,8 +232,6 @@ class BucketLocationConstraint(str):
     us_gov_west_1 = "us-gov-west-1"
     us_west_1 = "us-west-1"
     us_west_2 = "us-west-2"
-    ap_south_2 = "ap-south-2"
```

Comment:
damn, seems like my botocore was not updated. 😭 will open a PR reintroducing them? or wait for the next automatic ASF update.

Reply:
You can just manually trigger the action to trigger the update PR, which I did. The result is #9146. :)